modules/data.atmosphere/R/met.process.R

```diff
@@ -314,15 +317,17 @@ met.process <- function(site, input_met, start_date, end_date, model,
                                        host = host, 
                                        overwrite = overwrite$standardize,
                                        ensemble_name = i)
+        is_standardized <- TRUE
+      } else {
+        is_standardized <- FALSE
+      }
+
+      if(is_standardized) {
```

Comment:
I think this changes behavior: previously `ready.id` would have an entry for each id whether it was standardized or not; this only adds the ids that were standardized. Is that intentional?

(As currently implemented I think ids not standardized would probably have errored out earlier, but this step should not rely on that)

Reply:
I've implemented a `NA` appending too to make sure to append even when we have face a `!is_standardized` condition. The initial though process I had was whether it would be wise pass the `input.id`'s to `.met2model.module` and later on to `convert_input` if we have not standardised our results. Although it seems I failed and ignored the order in array at check which may change the results.

Reply:
@Sweetdevil144 I was wrong above about the existing behavior -- In the current code `standardize_result` has an entry for each id, but it is NULL for ids not standardized and therefore gets silently dropped when appending to `ready.id`. I don't know how advisable that is, but I bet downstream code relies on it.

I'll suggest changes here that eliminate the second for-loop but keep the original behavior -- please edit further if you see problems I missed.